release/models/policy/openconfig-routing-policy.yang

```diff
@@ -80,9 +80,21 @@ module openconfig-routing-policy {
     policy.  Similarly, a reject-route action returns false.  If the
     subroutine returns true, the calling policy continues to evaluate
     the remaining conditions (using a modified route if the
-    subroutine performed any changes to the route).";
+    subroutine performed any changes to the route).
 
-  oc-ext:openconfig-version "3.4.0";
+    If a routing protocol is defined and the related
```

Comment:
Does this also cover for Route redistribution polices using table-connections? https://github.com/openconfig/public/blob/master/doc/network_instance_redistribution.md. OC path for default-policy attachment on table-connections is /network-instances/network-instance/table-connections/table-connection/config/default-import-policy
I think we should clarify here that the default of reject_all is true for table-connections as well if no default-import-policy is defined.

Reply:
Yes, it's reject or all contexts where route policy is used.   The route policy model is used in many portions of openconfig and will get updated if/when new protocols are added.  We probably shouldn't try to list them all in the description here.

Reply:
does this apply to routing protocols other than BGP? the wording in 85 seems to be generic?

Reply:
Good question!  The defaults takes effect everywhere  routing policy is applied.  Today in OC, the following paths reference routing-policy:

/network-instances/network-instance/inter-instance-policies/apply-policy (between vrf's)
/network-instances/network-instance/protocols/protocol/bgp/global/apply-policy.  (bgp)
/network-instances/network-instance/table-connections/table-connection/config/import-policy (between protocols in a vrf)